scripts/check-orphans.py

```
@@ -0,0 +1,261 @@
+                # Additional filter: must be a valid agent-style name
+                if re.match(r"^[a-z]+-[a-z]+(?:-[a-z]+)?$", agent):
+                    orphans.append({
+                        "file": "CLAUDE.md",
+                        "reference": agent,
+                        "type": "routing-table"
+                    })
+
+    return orphans
+
+
+def check_agent_self_references(verbose=False):
```

Ruff (0.14.8)

Unused function argument: `verbose`

(ARG001)